lib/open_api_spex/open_api.ex

```diff
@@ -79,74 +83,76 @@ defmodule OpenApiSpex.OpenApi do
       defimpl encoder do
         def encode(api_spec = %OpenApi{}, options) do
           api_spec
-          |> to_json()
+          |> OpenApi.to_map
```

Comment:
Why do we need the `OpenApi` part?

```suggestion
          |> to_map
```